app/off.py

```diff
@@ -78,8 +79,19 @@ def wikidata(query, value):
         OSM_relation = "https://www.openstreetmap.org/relation/{}".format(osm)
     else:
         OSM_relation = ""
-
-    return (
+    Entities = namedtuple(
+        "Entities",
+        [
+            "label_tag",
+            "description_tag",
+            "image_url",
+            "entity_id",
+            "OSM_relation",
+            "INAO_relation",
+            "wikipedia_relation",
+        ],
+    )
+    E = Entities(
```

Comment:
```suggestion
    entities = Entities(
```
Here we got an instance of the class, a variable, so you should not use a capital letter.